### ISSUE TITLE
Better Info Cards v.2020.11.28.4, still crashing.

### ISSUE THREAD

**Issue by ragzilla** [AzeTheGreat/ONI-Mods#10]
Still seeing a crash in better info cards, in this instance it's related I'm triggering it with a large number of molten slicksters (151) in a 20 tile wide room.

https://gist.github.com/ragzilla/ecc996c7337eaf70297cb45bce4ae4dd

**Comment by AzeTheGreat** [comment]
Thank you for the report, I'll look into this.

By any chance, can you consistently (even somewhat) reproduce the issue here?

Also, did this actually crash the game?  This exception should theoretically be caught...

**Comment by ragzilla** [comment]
Crash is reproducible on my save with just "Better Info Cards" enabled. Wasn't able to repro on a fresh sandbox, so it seems to be linked to colony size/activity.  I can make the save available to you if it'd be useful.

**Comment by AzeTheGreat** [comment]
I want to thank you once again for being the first (and only) person to provide enough info to actually solve this bug.

I traced this down to an extremely rare vanilla bug that I have [now reported](https://forums.kleientertainment.com/klei-bug-tracker/oni/info-cards-displaying-blank-status-items-r28619/).

I have added fallback logic to handle this case in https://github.com/AzeTheGreat/ONI-Mods/commit/88b1f59ef69478c64bb786e2b401173d40af53cd, and have released an update.  This bug should now be solved.
